setup/unattended-install/InstallationFileService.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+require_once(APPROOT.'/application/utils.inc.php');
+require_once(APPROOT.'/setup/setuppage.class.inc.php');
+require_once(APPROOT.'/setup/wizardcontroller.class.inc.php');
+require_once(APPROOT.'/setup/wizardsteps.class.inc.php');
+
+if (version_compare(ITOP_DESIGN_LATEST_VERSION, '2.7', '<=')) {
+	require_once(APPROOT.'/core/config.class.inc.php');
+	require_once(APPROOT.'/core/log.class.inc.php');
+	require_once(APPROOT.'/core/kpi.class.inc.php');
+	require_once(APPROOT.'/core/cmdbsource.class.inc.php');
+	require_once(APPROOT.'/application/clipage.class.inc.php');
+}
```

Comment:
```suggestion
require_once(APPROOT.'/core/config.class.inc.php');
require_once(APPROOT.'/core/log.class.inc.php');
require_once(APPROOT.'/core/kpi.class.inc.php');
require_once(APPROOT.'/core/cmdbsource.class.inc.php');
require_once(APPROOT.'/application/clipage.class.inc.php');
```

Then during the 2.7 => 3.0 merge, remove these lines.
As this file is part of the iTop repo, we rather adapt the code in the branches rather than maintaining "ifs" (which we do in modules)

Reply:
ok i will keep it in mind